src/main/java/gregtech/integration/jei/recipe/RecipeMapCategory.java

```diff
@@ -133,7 +127,7 @@ public void setRecipe(IRecipeLayout recipeLayout, @Nonnull GTRecipeWrapper recip
                             new FluidStackTextRenderer(fluidAmount, false,
                                     tankWidget.getSize().width - (2 * tankWidget.fluidRenderOffset),
                                     tankWidget.getSize().height - (2 * tankWidget.fluidRenderOffset), null)
-                                    .setNotConsumed(notConsumedFluidInputs.get(importIndex)),
+                                    .setNotConsumed(recipeWrapper.isNotConsumedInput(importFluids.getFluidTanks().get(importIndex))),
```

Comment:
This probably is not getting the correct FluidStack, likely due to the same reason as above - the TankWidget does not actually contain a fluid